typescript/syntax/typescript.sdf3

```diff
@@ -3,10 +3,14 @@ module typescript
 imports
   
   Common
+ 
+template options
+
+  tokenize: "(,[;{.="
```

Comment:
We will address this later, as SDF3 does not support multiple characters as 1 token in the `tokenize` template option